Formula/tnftp.rb

```diff
@@ -0,0 +1,38 @@
+class Tnftp < Formula
+  desc "NetBSD's FTP client (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftp/lukemftp-16.tar.gz"
+  sha256 "ba35a8e3c2e524e5772e729f592ac0978f9027da2433753736e1eb1f1351ae9d"
+
+  depends_on :xcode => :build
+
+  conflicts_with "inetutils", because: "both install `ftp' binaries"
+
+  def install
+    # Trying to use Apple's pre-supplied Makefile resulted
+    # in headaches... they have made the build process
+    # specifically for installing to /usr/bin and so it
```

Comment:
Could always change that with `inreplace` if it's not too "involved".

Reply:
@DomT4 I considered doing something like that, but I concluded that it probably would be too involved. Plus, what I have there already works so I don't see any real issue there.